packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature

```diff
@@ -19,14 +19,15 @@ Feature: Tax And Cash Dashboard
 
   @minutia @ui
   Scenario Outline: The Tax Form header is displayed to the participant
-    Given they submitted a W9/W8-BEN/W8-BEN-E tax form
+    Given they submitted a tax form
+    And their required tax document type is <requiredDocumentType>
     Then the Tax Form header displays "<documentType> tax form"
 
-    Examples: 
-      | documentType |
-      | W9           |
-      | W8 BEN       |
-      | W8 BEN-E     |
+    Examples:
+      | requiredDocumentType | documentType |
+      | W9                   | W-9          |
+      | W8BEN                | W-8          |
```

Comment:
@Locrian24 : 'W8-BEN'

Reply:
`requiredDocumentType` refers to the value sent by the backend, which doesn't include dashes

Reply:
I see, thank you!